calculation-service/src/main/java/org/java/mentorship/calculation/tasks/GenerateWeeklyCashFlowReport.java

```diff
@@ -0,0 +1,16 @@
+package org.java.mentorship.calculation.tasks;
+
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+@Component
+public class GenerateWeeklyCashFlowReport {
+
+    // At 07:30 every 7 days
```

Comment:
Delete this comment since the frequency is configurable.
We can put a comment in the property file